### PR TITLE
myxer: init at 1.2.0

### PR DESCRIPTION
**Pull request by ErinvanderVeen**
###### Motivation for this change
Myxer (a good alternative to pavucontrol) was missing from nixpkgs. This PR contains 2 commits as Myxer was updated to 1.2.0 after this PR was made. The first commits inits at 1.1.3, the second updates to 1.2.0.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/myxer/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  buildInputs = [ libpulseaudio glib pango gtk3 ];
+
+  # Currently no tests are implemented, so we avoid building the package twice
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A modern Volume Mixer for PulseAudio";
+    homepage = "https://github.com/Aurailus/Myxer";
+    license = licenses.gpl3;
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/myxer/default.nix:34]: gpl3Only or gpl3Plus?

File: pkgs/applications/audio/myxer/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "myxer";
+  version = "1.1.3";
+
+  src = fetchFromGitHub {
+    owner = "Aurailus";
+    repo = pname;
+    rev = "${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/myxer/default.nix:18]:
```suggestion
    rev = version;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/myxer/default.nix (hunk 1 above)
Comment (ErinvanderVeen) [reply pkgs/applications/audio/myxer/default.nix]: Thank you! Changed to gpl3Only.

File: pkgs/applications/audio/myxer/default.nix (hunk 2 above)
Comment (ErinvanderVeen) [reply pkgs/applications/audio/myxer/default.nix]: Done!

**Comment by ErinvanderVeen** [comment]
Myxer 1.2.0 was released a few hours ago. I think it's best to add the update to 1.2.0 to this MR so I'll put it on draft until I have upgraded it.

**Comment by ErinvanderVeen** [comment]
I have updated the PR to include the update to 1.2.0 and updated the title of the PR to reflect this change.

**Comment by ErinvanderVeen** [comment]
@SuperSandro2000 Could I kindly request you take another look at this MR? I have incorporated all your suggestions.

**Pull request merged**